src/vivarium/component.py

```diff
@@ -606,7 +607,9 @@ def build_lookup_table(
             raise ConfigurationError(f"Data '{data}' must be a LookupTableData instance.")
 
         if isinstance(data, list):
-            return builder.lookup.build_table(data, value_columns=list(value_columns))
+            return builder.lookup.build_table(
+                data, value_columns=list(value_columns) if value_columns else ()
```

Comment:
Shouldn't this pass an empty list if value columns is None?

Reply:
doesn't really batter b/c build_table takes a Sequence (in fact it defaults to ())